src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1687,7 +1687,7 @@ public static function unsetConnectionResolver()
     /**
      * Get the default table name associated with the model.
      *
-     * @param self|null $class
+     * @param self|null  $class
```

Comment:
incorrect phpdoc. should be `class-string<self>|null` or just `string|null`

Reply:
I modified it this is old version kindly check this https://github.com/laravel/framework/pull/39701/files

Reply:
![image](https://user-images.githubusercontent.com/20116830/142624364-ca11a6ee-b956-43d9-8306-f8e8a51aeedd.png)